Source/Blazorise.Bootstrap/Styles/components/_bar.scss

```diff
@@ -1,12 +1,4 @@
-﻿.navbar-nav .nav-link:hover {
-    cursor: pointer;
-
-    *:not(:has(.disabled)):not(.disabled) {
-        cursor: pointer;
-    }
-}
-
-.nav .nav-link:hover {
+﻿.nav .nav-link:hover {
```

Comment:
This looks useless to me. 

- I didn't discover a structure where .nav and the .nav-link are direct descendants. Because the .nav-link should be place inside the BarItem (.nav-item). right?

- Also I didn't found the usage of the `.nav` (but it's bit hard to find) class as our navbars are using `navbar-nav`..
- Also link have the cursors by being a `<a>`...

Reply:
I think that rule was meant for the Tabs. Which have `nav` class. But I'm not 100% sure.

![image](https://github.com/user-attachments/assets/06917ed7-f8ab-427a-8b24-f61d9fb0a746)